plugins/PrivacyManager/Controller.php

```diff
@@ -183,6 +185,11 @@ public function privacySettings()
         $view = new View('@PrivacyManager/privacySettings');
 
         if (Piwik::hasUserSuperUserAccess()) {
+            $jsCodeGenerator = new TrackerCodeGenerator();
+            $file = new File(PIWIK_DOCUMENT_ROOT . '/' . $jsCodeGenerator->getJsTrackerEndpoint());
+
+            $view->trackerFileName = $jsCodeGenerator->getJsTrackerEndpoint();
+            $view->trackerWritable = !$file->hasWriteAccess();
```

Comment:
We need to remove the `!` otherwise it is inverted

Reply:
sure. forgot to push that 🙈